docs/tool_interface.rst

```diff
@@ -88,13 +88,13 @@ View-components definition
 
 ----
 
-.. _progression_bar-label:
+.. _progression-bar-label:
```

Comment:
If you change it like that, you need to look for all the :ref:, some of them are with `progression_bar-label`, I am personnally in favor of having  `<same as view/view-component rst file>-label`